content/articles/univariate-time-series-analysis-and-forecasting-with-arima-and-seasonal-arima-in-python/index.md

```diff
@@ -0,0 +1,505 @@
+
+### Table of contents
+- [Prerequisites](#prerequisites)
+- [How the ARIMA model works](#how-the-arima-model-works)
+- [what is a stationary time series?](#what-is-a-stationary-time-series)
+- [What is differencing?](#what-is-differencing)
+- [Components of the ARIMA model](#components-of-the-arima-model)
+- [How to check for stationarity](#how-to-check-for-stationarity)
+- [Augmented Dickey-Fuller test](#augmented-dickey-fuller-test)
+- [How SARIMA works](#how-sarima-works)
+- [Time series dataset](#time-series-dataset)
+- [Using the dataset](#using-the-dataset)
+- [Visualizing the time series data](#visualizing-the-time-series-data)
+- [Implementing Augmented Dickey-Fuller test](#implementing-augmented-dickey-fuller-test)
+- [Applying the function](#applying-the-function)
+- [Implementing differencing](#implementing-differencing)
+- [Plotting the new dataset](#plotting-the-new-dataset)
+- [Implementing the ARIMA model](#implementing-the-arima-model)
+- [Plotting ACF](#plotting-acf)
+- [Plotting PACF](#plotting-pacf)
+- [Import the ARIMA model](#import-the-arima-model)
+- [Training the ARIMA model](#training-the-arima-model)
+- [Testing the ARIMA model using the test dataset](#testing-the-arima-model-using-the-test-dataset)
+- [Implementing the SARIMA model](#implementing-the-sarima-model)
+- [Training the SARIMA model](#training-the-sarima-model)
+- [Testing the SARIMA model using the test dataset](#testing-the-sarima-model-using-the-test-dataset)
+- [Conclusion](#conclusion)
+- [References](#references)
+
+### Prerequisites
+To easily follow this tutorial, a reader should know the following:
+
+- [Time series concepts](/engineering-education/introduction-to-time-series/)
+- [Time series analysis and modelling.](/engineering-education/building-a-time-series-weather-forecasting-application-in-python/)
+- [Time series applications](https://www.tutorialspoint.com/time_series/time_series_applications.htm)
+- [Time series patterns](https://otexts.com/fpp2/tspatterns.html)
+- [Time series plots](https://towardsdatascience.com/5-types-of-plots-that-will-help-you-with-time-series-analysis-b63747818705)
+
+> NOTE: Ensure you implement the ARIMA and SARIMA models in [Gooogle Colab.](https://research.google.com/colaboratory/)
+
+### How the ARIMA model works
+Auto-Regressive Integrated Moving Average(ARIMA) is a time series model that uses the information in the past time series values to make future predictions. 
+
+The information found in the past values will indicate the nature of the future predictions. 
+
+For example, an ARIMA model can predict future stock prices after observing and analyzing previous stock prices.
+
+The ARIMA model will analyze the single time-dependent (univariate) variable in the times series. ARIMA models only work when the time series is stationary.
```

Comment:
As a rule of thumb, please ensure that your sentences are at least 3 lines long.